src/App.js

```diff
@@ -2,78 +2,133 @@
 import React, { useState, useEffect } from "react"; //import useState
 import "./App.css";
 import BottomRow from "./BottomRow";
-import HomeTouchBtn from "./components/homeTouchBtn";
+import soundFile from "./assets/air-horn-club-sample_1.mp3";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
 
   //home
-  const [homeTeam, setHome] = useState("Lions");
+  const [homeTeam, setHome] = useState("Raiders");
   const [homeScore, setHomeScore] = useState(0);
   //away
-  const [awayTeam, setAway] = useState("Tigers");
+  const [awayTeam, setAway] = useState("Bears");
   const [awayScore, setAwayScore] = useState(0);
 
+  //quarter
+  const [quarter, setQuarter] = useState(1);
+
   //timer
-  function Timer() {
-    const [time, setTimer] = useState("30:00");
-  }
+  const gameTime = 30;
+  const [time, setTimer] = useState(gameTime);
+  const [timeOn, setTimeOn] = useState(false);
+  const [playedSound, setPlayedSound] = useState(false);
+
+  const [playSound, setPlaySound] = useState(0);
+
+  const quarterFunc = () => {
+    if (quarter < 4) {
+      setQuarter(quarter + 1);
+    } else setQuarter(1);
+  };
+
+  const toggle = () => {
+    setTimeOn(!timeOn);
+  };
+
+  useEffect(() => {
+    let interval = null;
+    if (timeOn && time >= 1) {
+      interval = setInterval(() => {
+        setTimer(seconds => seconds - 1);
+      }, 1000);
+    } else if (!timeOn && time !== 0) {
+      clearInterval(interval);
+    } else if (time === 0) {
+      if (!playedSound) {
+        setPlayedSound(true);
+        setPlaySound(!playSound);
+        toggle();
+        setTimeout(() => {
+          setTimer(gameTime);
+        }, 1000);
+      }
+    }
+    return () => {
+      clearInterval(interval);
+    };
+  }, [timeOn, time]);
```

Comment:
This effect is getting quite big. I would have liked to see this broken down int multiple effects. One effect for the timer and one effect for playing the sound.